origin.spec

```diff
@@ -50,6 +50,7 @@ Obsoletes:      openshift < 1.0.6
 %package master
 Summary:        %{product_name} Master
 Requires:       %{name} = %{version}-%{release}
+Requires:       %{name}-clients = %{version}-%{release}
```

Comment:
Maybe add it as a dependency of the base package too so that if they install `origin` they get the full suite of binaries even if they're not installing the systemd units?